core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -159,6 +161,19 @@ public Runner getActiveRunner() {
       /* Number of Kafka partitions to to use for managed feature stream. */
       @Positive private int partitions = 1;
     }
+
+    @Getter
+    @Setter
+    public static class FeatureSetSpecStreamProperties {
+      /* Kafka topic to send feature set spec to ingestion streaming job */
+      @NotBlank private String specsTopic = "feast-feature-set-specs";
+
+      /* Kafka topic to receive acknowledgment from ingestion job on successful processing of new specs */
+      @NotBlank private String specsAckTopic = "feast-feature-set-specs-ack";
+
+      /*Comma separated list of Kafka bootstrap servers */
+      @NotBlank private String bootstrapServers = "localhost:9092";
```

Comment:
Will we have a use case for splitting the spec stream and feature stream? I think we can maybe roll these options into a single class.